docs/content/en/integrations/importing.md

```diff
@@ -7,13 +7,13 @@ weight: 1
 
 ## Import
 
-The importers analyze each report and create new Findings for each item
-reported. DefectDojo collapses duplicate Findings by capturing the
+The importers analyze each report and create new findings for each item
+reported. DefectDojo collapses duplicate findings by capturing the
 individual hosts vulnerable.
 
 ![Import Form](../../images/imp_1.png)
 
-This approach will create a new Test for each upload. This can result a lot of findings. If deduplication is enabled, new findings that are identical to existing findings get marked as a duplicate.
+This approach will create a new Test for each upload. This can result in a lot of findings. If deduplication is enabled, new findings that are identical to existing findings get marked as a duplicate.
```

Comment:
```suggestion
This approach will create a new test for each upload. This can result in a lot of findings. If deduplication is enabled, new findings that are identical to existing findings get marked as a duplicate.
```